README.md

```diff
@@ -84,6 +84,19 @@ Read about the design here:
 - [Middleware in Tide](https://rustasync.github.io/team/2018/11/07/tide-middleware.html)
 - [Tide's evolving middleware approach](https://rustasync.github.io/team/2018/11/27/tide-middleware-evolution.html)
 
+### Supported Rust Versions
+
+Tide is built against the latest Rust nightly releases and as such, due to it's use of `std` futures,
+it has the following specific breakpoints that align with std future API changes:
+
+Tide | Rust
+--- | ---
+&le; v0.1.0 | &le; nightly-2019-04-07
+&ge; v0.1.1 | &ge; nightly-2019-04-08
```

Comment:
I'm liking this a lot, but because we're still actively developing + futures aren't stable yet, I think we could get away with only keeping a single number around. I guess I'm wondering how useful this will still be after, say, 10 releases.

Or do you think we should keep a habit of tracking these numbers?

Reply:
:100 with you. I was actually thinking 2-3 releases up ahead, we can trim 0.1.0 to a small note instead - and then completely remove it having after a few more releases, after which this section will likely be just one or two lines.